docs/source/_static/custom.css

```diff
@@ -1,3 +1,8 @@
 td {
     padding: 0px 10px 0px 10px;
 }
+
+img.align-center {
+    width: 100%;
+    margin-bottom: 10px;
```

Comment:
I consider this should be padding and should be on all sides.
```suggestion
    padding: 10px;
```

Reply:
No, I actually wanted to add `margin-bottom`, same as for the surrounding paragraphs so that the spacing is symmetrical.

Reply:
Ok then it need some padding ( Space BTN text and images) though. So add both.

Reply:
Also I'm not on PC right now so screenshot of how it looks is appreciated.

Reply:
Here is how it currently looks:

![image](https://user-images.githubusercontent.com/11851593/91753517-49101a00-ebc8-11ea-9228-3f61886b7ea4.png)

The space between text and image is the same as between two paragraphs.

The alternative would be to have the same spacing as code blocks, which would be `20px` margin on top and bottom, what do you think about that?

If you want to look at it on mobile, I have deployed the current status here: https://manim.hackl.space/tutorials/quickstart.html

Reply:
Yes the 20 is ok.